app/assets/javascripts/client/inventories/productEntryModalCtrl.js

```diff
@@ -7,17 +7,18 @@
   ProductEntryModalCtrl.$inject = [
```

Comment:
Convention:  Since this is a controller, the file name should start with a capital.